### ISSUE TITLE
Binding for residue-residue structural alignment 

### ISSUE THREAD

**Issue by kamurani** [jvkersch/tmtools#20]
First of all, thanks for creating this package.  

I was looking for a way to extract the "residue-residue" alignment that TM-align generates before it finds the optimal alignment for all-to-all residue comparisons across the entirety of the two input structures.  

Basically what I would like to do is calculate the TM score for two structures, *given* that they are superimposed at a particular position (i.e. residue position x from structure 1 is overlayed to residue position y in structure 2).

The rationale behind this is to "impose" a specific known site that is analogous between two protein structures and get the alignment score (which would be optimised by rotating the structures while maintaining their translation such that residues x and y are fixed in the same position).  

i.e. given that two structures are overlayed at `structure_1, residue_x` and `structure_2, residue_y`, what is the structure-structure TM score? 

As far as I am aware, this sort of information is generated in the TM-align algorithm (the dynamic programming component) before the "best" residue-residue mapping is picked to optimise the score.  Do you know if it would be easy to create a python binding to return this level of information, given two structures and a specific residue from each?

Or any other methods to get this "fixed residue pair" score? 

Thanks in advance.

**Comment by jvkersch** [comment]
@kamurani Do you happen to know where exactly in the code this is done? I understand (on a high level) what you want to do, but I'm afraid I'm not sufficiently familiar with the code to understand the details or the implications.

**Comment by kamurani** [comment]
@jvkersch No, I am pretty unfamiliar with the TM algorithm and haven't looked too deeply at it yet.  

I am assuming that there must be a process in which the 'optimal rotation' for a *particular* translation is worked out; so in principle I would be able to find this in the code and simply perform this routine once (for the translation I impose, i.e. residue x and residue y are both at `(0,0,0)`). 

To achieve what I want to do, I have looked at other methods (e.g. the [Kabsch algorithm](https://en.wikipedia.org/wiki/Kabsch_algorithm), which finds the optimal rotation (and can be used to give an RMSD score) *given* a particular translation of points) - but these require the same number of points in each structure (but TM align can work with differing protein lengths).
